### PR TITLE
Add support for password grant oauth flow

### LINKED ISSUE #2081: Auth Clients Grant Secret
Show the password grant client secret in the Auth Clients screen in administration

This needs to be visible so that people can use the password grant type effectively.

### PR DESCRIPTION
**Pull request by nolanpro**
Fixes #2081 

image

### CODE REVIEW

File: resources/views/admin/auth-clients/index.blade.php [hunk 1]
```diff
@@ -53,6 +52,13 @@
 
         <div class="px-3 page-content">
             <div class="row">
+                <div class="col" align="left">
```
Comment (caleeli) [inline resources/views/admin/auth-clients/index.blade.php]: Why are showing above the table only the first client with password grant enabled?
@velkymx @nolanpro should be better add a checkbox to enable the Password Grant type in the OAuth Client Screen, something like this:

image

similar to other OAuth applications:
image

**Comment by velkymx** [comment]
I agree. @nolanpro please go ahead and make these changes.

**Comment by caleeli** [comment]
@nolanpro, validation message is not shown to the end user when save client with no grants assigned.

image

**Comment by nolanpro** [comment]
@caleeli ok added that

**Pull request merged**